MOxUnit/assertEqual.m

```diff
@@ -26,14 +26,14 @@ function assertEqual(a, b, message)
 % NNO Jan 2014
 
 
-    if ~isequal(size(a), size(b))
-        whatswrong='inputs are not of the same size';
-        error_id='assertEqual:nonEqual';
-
-    elseif ~isequal(class(a), class(b))
+    if ~isequal(class(a), class(b))
         whatswrong='inputs are not of the same class';
         error_id='assertEqual:classNotEqual';
 
+    elseif ~isequal(size(a), size(b))
```

Comment:
See commit message.
>>>
If the classes are different, this is a more fundamental problem
than the sizes being different and so it is more useful to know
this instead.